src/draft-js-utils.ts

```diff
@@ -81,3 +82,17 @@ export function replaceHandle(editorState: EditorState, handle: string): EditorS
 
   return prependHandle(fromText(restOfTheText), handle)
 }
+
+export function logErrorAndReturnAlert(alert) {
+  console.log(`Error name: ${alert}`)
+
+  if (alert === 'SyntaxError') {
+    window.alert(
```

Comment:
The idea is for the error to be logged as is to the console using [console.error](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) and then to inspect the error and return a human readable alert. The rationale is that we want a developer to be able to see the error as it was returned by the machine, but for the user to see an alert that is human readable to them.

We should not use [window.alert](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert) which renders the alert in a dialog box because we want to render our stylized react component instead and because `window.alert` halts any running javascript until the user clicks the OK button.

this is that being that error messages may not be human readable on there



The idea is for us to log the error to the console using `console.error`

Reply:
I would expect an implementation of this function to look like something like this

```typescript
export function logErrorAndReturnAlert(error: unknown) {
  console.error(error)

  if (someTestToTellIfThisIsANetworkError(error)) {
    return 'Could not connect to the server. Please check your connection and try again.'
  }

  ...other error tests with specific error messages...

  return 'An error occurred. Please try again. If the error persists, please contact support@morehumaninternet.org'
}
```